test-case/verify-kernel-boot-log.sh

```diff
@@ -21,4 +21,6 @@ source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
 func_opt_parse_option "$@"
 disable_kernel_check_point
 
+print_module_params
+
```

Comment:
Considering the output is short enough I think it's tempting to run this somewhere in `case-lib/lib.sh` so it runs for every test.

Reply:
For SOF CI purpose, we don't have any use case to change any module parameter. I think once in first test case is good enough.